tests/framework/extensions/provisioninginput/config.go

```diff
@@ -37,47 +37,59 @@ const (
 )
 
 var AllRolesMachinePool = MachinePools{
```

Comment:
With these changes, can you make sure that both the RKE2/K3S READMEs are updated with this updated config? That way, we ensure everything is updated.